models/labels/miners/labels_miners.sql

```diff
@@ -0,0 +1,66 @@
+{{config(alias='miners',
+        post_hook='{{ expose_spells(\'["ethereum", "arbitrum", "gnosis", "optimism", "bnb", "avalanche_c"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["soispoke"]\') }}')
+}}
+
+SELECT array('ethereum') as blockchain,
```

Comment:
do these statements need a DISTINCT? 

there's multiple rows in arbritum.blocks per miner from what I see. 
https://dune.com/queries/1285087?d=1

Reply:
@dot2dotseurat Yes, thanks good catch !
(now fixed)